halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Telemetry.java

```diff
@@ -33,7 +33,7 @@ public String getNodeName() {
 
   private Boolean enabled = false;
   private String endpoint = DEFAULT_TELEMETRY_ENDPOINT;
-  private String instanceId = UUID.randomUUID().toString();
+  private String instanceId = new ULID().toString();
```

Comment:
I'm pretty sure this is just serializing the `ULID` generator itself, and you need to to call `nextULID()` on the generator to get an actual ULID.

Reply:
That's what I get for trying to move too fast. Fixed.